edenia/eden-spend-explorer
# feat(webapp): improve homepage

### Improve homepage

### What does this PR do?

- Resolve #198 

### Steps to test

1. run the project and open localhost:3000 in your browser
2. click on home in the sidebar and see the new features

#### CheckList
